src/main/java/teamproject/TestData.java

```diff
@@ -0,0 +1,13 @@
+package teamproject;
+
+public interface TestData {
+	//This class is a placeholder with hardcoded data.
+	String[][] test = {
+		{"Scrambled Eggs", "Easy"},
```

Comment:
The current plan is to have a "recipe" be represented by a JSON object that looks like this:
```
{
"id": 0,
“link”: “some link”,
“title”: “some title”,
“likes”: 5,
“category”: “some category”,
}
```

It'll be easier to swap to fetching from DataStore later on if you make your test data look more like that. I'd recommend using a HashMap to represent the JSON blobs for now, and having an array of those. So, something like:
```
Hashmap<String, String>[] test = {
    {
        "id": "0",
        "link": "test.com",
        ...
    },
    {
        "id": "1",
        "link": "example.com",
        ...
    }
    ...
}
    
```

Reply:
Having difficulty using HashMap. Java throws an error indicating that I can't make a HashMap array. Tried using a list instead, and was unable to get it working either. Should I just use a String array for the time being?